clients/ranchercli/client.go

```diff
@@ -0,0 +1,92 @@
+package ranchercli
+
+import (
+	"fmt"
+	osys "os"
+	"os/exec"
+	"strings"
+
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/sirupsen/logrus"
+)
+
+type Client struct {
+	Session *session.Session
+}
+
+func (c *Client) Create(name string, args ...string) error {
+	return c.ExecuteCommand(name, args...)
```

Comment:
why do we want to have the Create function calling the ExecuteCommand function? Can we not have just the ExecuteCommand?

Reply:
Resolved, spoke offline regarding this.